src/transformers/models/whisper/feature_extraction_whisper.py

```diff
@@ -94,41 +94,57 @@ def __init__(
             mel_scale="slaney",
         )
 
-    def _np_extract_fbank_features(self, waveform: np.array) -> np.ndarray:
+    def _np_extract_fbank_features(self, waveform_batch: np.array) -> np.ndarray:
         """
         Compute the log-mel spectrogram of the provided audio, gives similar results to Whisper's original torch
         implementation with 1e-5 tolerance.
```

Comment:
let's update the doc here!

Reply:
I have added some changes, but I'm unsure if it's correct. @ArthurZucker